.circleci/config.yml

```diff
@@ -37,49 +37,61 @@ jobs:
                     codecov
 
     integration-test:
-        docker:
-            - image: docker:18.05.0-ce-git
-
+        machine: true
```

Comment:
is there a reason this is using machine instead of container?

Reply:
ah, I see you're just hacking around it to find out why things are failing :)

Reply:
Yup. While the acls work on a 'real' machine, the filesystem on CI has some of the same acl issues that images have. After spending a bit of time, I think we can't reasonably test on Circle if we want to see how a real filesystem would be have.